group.yml

```
@@ -280,11 +280,10 @@ repos:
   rhel-9-baseos-rpms:
     conf:
       baseurl:
-        # NOTE: re-peg to 9.0 when 9.1 GA is pending
-        aarch64: http://rhsm-pulp.corp.redhat.com/content/dist/rhel9/9/aarch64/baseos/os/
-        ppc64le: http://rhsm-pulp.corp.redhat.com/content/dist/rhel9/9/ppc64le/baseos/os/
-        s390x: http://rhsm-pulp.corp.redhat.com/content/dist/rhel9/9/s390x/baseos/os/
-        x86_64: http://rhsm-pulp.corp.redhat.com/content/dist/rhel9/9/x86_64/baseos/os/
+        aarch64: http://rhsm-pulp.corp.redhat.com/content/dist/rhel9/9.0/aarch64/baseos/os/
+        ppc64le: http://rhsm-pulp.corp.redhat.com/content/dist/rhel9/9.0/ppc64le/baseos/os/
+        s390x: http://rhsm-pulp.corp.redhat.com/content/dist/rhel9/9.0/s390x/baseos/os/
+        x86_64: http://rhsm-pulp.corp.redhat.com/content/dist/rhel9/9.0/x86_64/baseos/os/
```

We want to use EUS repos in order to get ongoing updates:
```suggestion
        aarch64: http://rhsm-pulp.corp.redhat.com/content/eus/rhel9/9.0/aarch64/baseos/os/
        ppc64le: http://rhsm-pulp.corp.redhat.com/content/eus/rhel9/9.0/ppc64le/baseos/os/
        s390x: http://rhsm-pulp.corp.redhat.com/content/eus/rhel9/9.0/s390x/baseos/os/
        x86_64: http://rhsm-pulp.corp.redhat.com/content/eus/rhel9/9.0/x86_64/baseos/os/
```